### PR TITLE
Updated CHANGELOG for 0.5.0

### PR DESCRIPTION
**Pull request by wjzijderveld**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+
+##### Bundle
+
+- The global `storage_suffix` parameter has been removed and has been replaced with a configuration value: `saga.mongodb.storage_suffix`.
+
+#### Summary of other changes
+
+- New example on how to use child entities
+- The EventSourcing Scenario has been updated to support all the latest changes. 
+- A AggregateRootScenarioTestCase has been added with an example on how to use it
```
Comment (asm89) [inline CHANGELOG.md]: nit: An

**Comment by asm89** [comment]
:+1:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+##### Bundle
+
+- The global `storage_suffix` parameter has been removed and has been replaced with a configuration value: `saga.mongodb.storage_suffix`.
+
+#### Summary of other changes
+
+- New example on how to use child entities
+- The EventSourcing Scenario has been updated to support all the latest changes. 
+- An AggregateRootScenarioTestCase has been added with an example on how to use it
+- The command bus and event bus have been made more resilient
```
Comment (cordoval) [inline CHANGELOG.md]: be consistent with dots at the end of the phrases

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v0.5.x
+
+#### BC breaks
```
Comment (sstok) [inline CHANGELOG.md]: This will make it header h4, should be `###` to be consistent :)

Comment (wjzijderveld) [inline CHANGELOG.md]: Consistent with what? As I use 4 everywhere :smile: 
It is personal preference, but I find ### still to big :)

**Comment by rjkip** [comment]
@wjzijderveld Can this be merged? Had to find out about the BC breaks post-update :'(

**Comment by wjzijderveld** [comment]
Sorry @rjkip :disappointed:

**Comment by mbadolato** [comment]
@wjzijderveld Does this need a 0.5.1 tag? It won't be available if someone's composer is set to ~0.N otherwise and they would have the same BC issues @rjkip had

**Comment by rjkip** [comment]
You are forgiven :wink:

**Pull request merged**